_api-reference/index-apis/update-alias.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: default
+title: Create or update alias
+parent: Index APIs
+nav_order: 5
+---
+
+# Create or update alias
+**Introduced 1.0**
+{: .label .label-purple }
+
+The Create or update alias API adds a data stream or index to an alias or updates the settings for an existing alias. For more Alias API operations, see [Index aliases]({{site.url}}{{site.baseurl}}/opensearch/index-alias/).
+
+The Create of update alias API is distinct from the [Alias API]({{site.url}}{{site.baseurl}}/opensearch/rest-api/alias/), which supports the adding and removal of aliases and the removal of alias indexes. In contrast, the following API only supports adding or updating an alias without updating the index itself. Each API also uses different request body parameters.
+
+The which supports adding aliases, removing aliases and removing indexes, this API only supports adding or updating an alias, the URL path and request body parameters of these two APIs are also different, so be careful to use them.
+{: .note}
+
+## Path and HTTP methods
+
+```
+POST /<target>/_alias/<alias-name>
+PUT /<target>/_alias/<alias-name>
+POST /_alias/<alias-name>
+PUT /_alias/<alias-name>
+POST /<target>/_aliases/<alias-name>
+PUT /<target>/_aliases/<alias-name>
+POST /_aliases/<alias-name>
+PUT /_aliases/<alias-name>
+PUT /<target>/_alias
+PUT /<target>/_aliases
+PUT /_alias
+```
+
+## Path parameters
+
+| Parameter | Type | Description |
+:--- | :--- | :---
+| `target` | String | A comma-delimited list of data streams, indexes. Wildcard expressions (`*`) are supported. To target all data streams and indexes in a cluster, use `_all` or `*`. Optional. |
+| `alias-name` | String | The alias name to be created or updated. Optional. |
+
+## Query parameters
+
+All query parameters are optional.
+
+Parameter | Type | Description
+:--- | :--- | :---
+`cluster_manager_timeout` | Time | The amount of time to wait for a response from the cluster manager node. Default is `30s`.
+`timeout` | Time | The amount of time to wait for a response from the cluster. Default is `30s`.
+
+## Request body
+
+In the request body, you can specify the index name, the alias name, and the settings for the alias. All fields are optional.
+
+Field | Type | Description
+:--- | :--- | :--- | :---
+`index` | String | Comma-delimited list of data streams or indexes you want to associate with the alias. If this field is set, it will override the index name specified in the URL path.
```

Comment:
```suggestion
`index` | String | A comma-delimited list of data streams or indexes that you want to associate with the alias. If this field is set, it will override the index name specified in the URL path.
```